### PR TITLE
fix: e2e run

### PR DESCRIPTION
**Pull request by debendraoli**
### Commit Message

```bash
fixes: self hosted running for e2e
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

E2E can be executed using PR comment.

Example:
```bash
/run e2e 1.2
```
Where 1.2 is a relayer version to use.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [x] I only have one commit (if not, squash them into one commit).
- [x] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)
- [ ] I have added version bump label on PR.

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: .github/workflows/e2e-runner.yml [hunk 1]
```diff
@@ -32,15 +42,8 @@ jobs:
...
-
-      - name: Set up Go 1.21
-        uses: actions/setup-go@v4
-        with:
-          go-version: 1.21
-
       - name: Run e2e tests
-        run: make e2e
-
+        run: ./scripts/execute-test.sh --clean --use-docker --build-ibc --build-xcall --xcall-branch main --test e2e
```
Comment (redlarva) [inline .github/workflows/e2e-runner.yml:46]: also can we use 'xcall-branch' based on the argument value?

**Review by AntonAndell — COMMENTED** [review]
Test case looks good, but how are they guaranteed to be on the same height and not to neighboring heights? @redlarva

**Comment by redlarva** [comment]
> guaranteed

Yes, you're correct, there's no guarantee that the packets will be sent at exactly the same time. However, since we're using a Go routine and it doesn't wait for a response to execute another round, it's likely that it will send the two packets with minimal time difference. I manually checked the packet height after running this test, and it appears that they are indeed at the same height.

**Review by redlarva — APPROVED** [review]
looks good to me

**Pull request merged**